Source/WebKit/UIProcess/Cocoa/WebPageProxyCocoa.mm

```diff
@@ -956,6 +956,14 @@
 
 #endif
 
+bool WebPageProxy::useGPUProcessForDOMRenderingEnabled() const
+{
+    if ([[[NSUserDefaults standardUserDefaults] objectForKey:@"WebKit2GPUProcessForDOMRendering"] boolValue])
```

Comment:
This doesn't work if you want `@NO` to override the `preferences()` value, so you should probably test for the existence of the `objectForKey` result before possibly calling `-boolValue` on nil.

Reply:
No it doesn't. But neither does the existing use of `WebKit2UseRemoteLayerTreeDrawingArea`. I can make them both handle `NO`, though the command line argument also doesn't support turning the flag off.

Reply:
Though I guess that one's different because there's no feature it's overriding.